icons/captions-off.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-captions-off"><path d="M7 15h4M7 11h2M2 2l20 20"/><path d="M17 11h-1M19 19H5s-2 0-2-2V7s-.28-2 2-2M21 16V7s0-2-2-2h-9"/></svg>
```

Comment:
This file need some formatting, see other icons as example.
Or see the lining errors to guide you.